javascript/javascript1/week3/SeriesDuration.js

```diff
@@ -0,0 +1,55 @@
+const seriesDurations = [
+    {
+      title: "Game of thrones",
+      days: 3,
+      hours: 1,
+      minutes: 0,
+    },
+    {
+      title: "Breaking Bad",
+      days: 4,
+      hours: 23,
+      minutes: 30,
+    },
+    {
+      title: "House",
+      days: 10,
+      hours: 9,
+      minutes: 18,
+    },
+  ];
+const eightyYearsInDays = 80*365;
+const onePercentInMinutes = ((eightyYearsInDays/100)*24)*60;
+function logOutSeriesText() {
+    function logOutFirstSerie() {
```

Comment:
Do you think you could have a universal funcion that would work for all three series? The only diffrence in this 3 is wich element we take from `seriesDuration` array. Maybe we could use map? (I know that `Array.map` came after this homeowork :) ) or just have an index as a parameter of it